### DISCUSSION TITLE
SleekwareDB - Open Source [HELP WANTED]

### DISCUSSION THREAD

**Discussion by darkterminal** [EddieHubCommunity/support#3953] [Show and tell]
With this post, I would like to say a big thank you to all EddieHubCommunity members who have taught me many things even though it's only been a few days.

Thanks to EddieHubCommunity I finally got the courage to publish the SleekwareDB project I built. Although it is not finished yet, because it is still under construction and requires contributions from community friends to be able to help run this project in the future.

Through [SleekwareDB Series on DEV.to](https://dev.to/darkterminal/sleekwaredb-open-source-5b6k) I will continue to update processes that occurred while building SleekwareDB.

- Github Project: https://github.com/SleekwareDB/sleekwaredb
- Discord: https://discord.gg/R35CEJPz

**Comment by SaptarshiSarkar12** [comment]
Great! All please show support.

**Reply by darkterminal** [reply]
Thanks you... I need more discussion and advice about this project
